modules/python/src/custom_nodes/google_genmedia/lyria2_api.py

```diff
@@ -92,14 +95,16 @@ def generate_music_from_text(
         if seed > 0:
             instance["seed"] = seed
             instance["sample_count"] = 1
-            print("Lyria Node: Seed is greater than 0, setting sample_count to 1.")
+            logger.info(
+                "Lyria Node: Seed is greater than 0, setting sample_count to 1."
+            )
         else:
             instance["sample_count"] = sample_count
-        print(f"Lyria Node: Instance: {instance}")
+        logger.info(f"Lyria Node: Instance: {instance}")
         response = self.client.predict(
             endpoint=self.model_endpoint, instances=[instance]
         )
-        print(
+        logger.info(
```

Comment:
The `Lyria Node: ` tag is redundant as the class should be included in the log message.